### PR TITLE
Cleanup datasets tutorial

### PR DESCRIPTION
**Pull request by AtreyeeS**
Please ignore if already addressed

### CODE REVIEW

File: examples/tutorials/api/datasets.py [hunk 1]
```diff
@@ -477,9 +473,9 @@
 
 
 ######################################################################
-# For an example of fitting `~gammapy.estimators.FluxPoints`, see
-#  :doc:`/tutorials/analysis-1d/sed_fitting`, and can be used for
-# catalog objects, e.g. see :doc:`/tutorials/api/catalog`
+# 
+# For an example of fitting `~gammapy.estimators.FluxPoints`, see :doc:`/tutorials/analysis-1d/sed_fitting`,
+# and for using source calalogs see :doc:`/tutorials/api/catalog`
```
Comment (registerrier) [inline examples/tutorials/api/datasets.py:478]:
```suggestion
# and for using source catalogs see :doc:`/tutorials/api/catalog`
```

File: examples/tutorials/api/datasets.py [hunk 2]
```diff
@@ -581,7 +577,3 @@
 
 datasets_sliced = datasets.slice_by_energy(energy_min="1 TeV", energy_max="10 TeV")
 print(datasets_sliced.energy_ranges)
-
-# %%
-
-plt.show()
```
[the remark this answers is not in the document]
Comment (AtreyeeS) [reply examples/tutorials/api/datasets.py]: Done

**Comment by registerrier** [comment]
Hi @AtreyeeS, please check the black format

**Review by registerrier — APPROVED** [review]
Thanks @AtreyeeS !

**Pull request merged**